samples/csharp_dotnetcore/06.using-cards/Startup.cs

```diff
@@ -37,7 +37,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<MainDialog>();
 
             // Create the bot as a transient. In this case the ASP Controller is expecting an IBot.
-            services.AddTransient<IBot, DialogAndWelcomeBot<MainDialog>>();
+            services.AddTransient<IBot, RichCardsBot<MainDialog>>();
```

Comment:
I'm ok with the name change - however now its clearly more specifc how about hard coding it to MainDialog and skipping the type parameterization (on the derived type)

Reply:
The type parameterization is now removed